traefik/docker-compose-traefik.yml

```diff
@@ -0,0 +1,27 @@
+##############################
+# FILENAME: docker-compose.yml
+# DESCRIPTION: This file is used to configure the docker for Traefix HTTP Reverse Proxy Services
+# USAGE: The file should be placed in /opt/traefik directory. 
+#        It is important that the volume for the SSL Certificates be added and that they correspond with the traefik.toml file in this directory.
+#	   
+##############################
+
+version: '3.4'
+
+services:
+  traefik:
+    image: traefik:latest
+    ports:
+      - 80:80
+      - 443:443
+      - 8080:8080
+    networks:
+      - traefik-net
+    volumes:
+      - /var/lib/boot2docker/certs:/var/lib/boot2docker/certs
```

Comment:
Do we need any named volumes, or are we good with simply having volumes mapped through from the host?

Reply:
So are you thinking about this instead:

--mount source=/var/lib/boot2docker/certs,target=/var/lib/boot2docker/certs \

Reply:
We may not need it if we don't have to persist data if the container gets removed.

Reply:
Because we are currently using Let's Encrypt we will want updated certs to be reflected to the running services.